conda/recipes/libkvikio/recipe.yaml

```diff
@@ -99,6 +101,7 @@ outputs:
                 - libcufile ${{ cuda11_libcufile_run_version }}
                 - libcufile-dev ${{ cuda11_libcufile_run_version }}
           else:
+            - cuda-nvml-dev
             - if: linux
```

Comment:
This line is a little odd. I don't think we ship kvikio on non-Linux platforms. However it's not changed in this PR so let's leave it alone. I am fine with adding `cuda-nvml-dev` outside this `if` block as it currently is.